data/json/mapgen/post_office.json

```diff
@@ -5,60 +5,52 @@
     "om_terrain": [ "post_office" ],
     "weight": 100,
     "object": {
+      "fill_ter": "t_floor",
       "rows": [
         "ssssssssssssssssssssssss",
         "g__________s||-------||g",
-        "g__________s|.........|g",
+        "g__________s|^.......Y|g",
         "gp____p____s-b.......#|g",
         "gp____p____s-b.......#|g",
-        "gp____p____s|........#|g",
-        "gp____p____s+.........|g",
+        "gp____p____s|Y.......#|g",
+        "gp____p____s+........2|g",
         "gp____p____s+........#|g",
         "gsssssssssss|&.......#|g",
         "g||w|||||||||v.......#|g",
-        "g|.....|...&|......||||g",
-        "g|..........|.......XX|g",
+        "g|^IIIF|...&|......||||g",
+        "g|..c.......|.......XX|g",
         "g|..........#........X|g",
-        "g|.....|...c#.....b..X|g",
-        "g|.....|....#.....b..X|g",
-        "g|#...X|....#........X|g",
-        "g|#...X|....|.......XX|g",
-        "g|#####|....|......||||g",
-        "g||||||||..||.........|g",
-        "g|f.........|^bb......|g",
-        "g|#.........|||||d|||D|g",
-        "g|...c......a......|T.|g",
-        "g|{{.tt.....|......|S.|g",
+        "4|F....|...c#.....b..X|g",
+        "g|F....|#...#.....b..X|g",
+        "g|#...X|#...#.....Y..X|g",
+        "g|#.K.X|#...|.......XX|g",
+        "g|#####|#...|......||||g",
+        "g||||||||..||......D.S|g",
+        "g|f%%%2.....|^bb..Y|.T|g",
+        "g|#.........|||||d|||||g",
+        "g|...c......a......D.T|g",
+        "g|{{.III.12.|.....Y|.S|g",
         "g||||||||||||||||d|||||g"
       ],
       "terrain": {
-        "#": "t_floor",
-        "&": "t_floor",
         "+": "t_door_glass_c",
         "-": "t_wall_glass",
         ".": "t_floor",
         "D": "t_door_locked",
-        "S": "t_floor",
-        "T": "t_floor",
-        "X": "t_floor",
-        "^": "t_floor",
         "_": "t_pavement",
         "a": "t_door_locked_interior",
-        "b": "t_floor",
-        "c": "t_floor",
         "d": "t_door_metal_locked",
```

Comment:
With no console nearby that unlocks these doors, it becomes almost impossible to open them (they ain't pickable or pryable). Is this indended?